tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -548,10 +548,6 @@ tf_module {
     name: "python_io"
     mtype: "<type \'module\'>"
   }
-  member {
```

Comment:
I don't think this should be deleted. Could you please keep it?

Reply:
@penpornk I did not remove anything manually. The golden tool removed it. Still want me to edit it manually?

Reply:
@penpornk anyways Done!